docs/semgrep-secrets/glossary.md

```diff
@@ -0,0 +1,90 @@
+---
+slug: glossary
+description: Definitions of Semgrep Secrets product-specific terms.
+tags:
+    - Semgrep Secrets
+title: Secrets glossary
+hide_title: true
+---
+
+# Semgrep Secrets glossary
+
+The terms and definitions provided here are specific to Semgrep Secrets.
+
+## Entropy analysis
+
+Entropy, the measure of a string's randomness, measures how likely it is that a given string is random. If a string is highly entropic, it's highly random. Entropy analysis, therefore, can provide insight into whether a given string is a secret, reducing false positives.
+
+## Expiration
+
+Some secrets are time-limited. This means that the secret is only valid for the period set by the creator. Expired secrets can pose fewer problems, so findings involving expired secrets can be deprioritized. 
+
+## Historical scan
+
+A scan of your Git commit history to see if there are valid secrets publicly available in your repository's Git history.
+
+## Policy
+
+A policy defines the set of rules that Semgrep runs and the workflow actions it undertakes when a rule from the policy generates a finding. The workflow action performed by Semgrep when it detects a finding can include notifying Slack channels or posting a comment in the pull request or merge request that generated the finding.
+
+## Registry (Semgrep Registry)
+
+A [<i class="fas fa-external-link fa-xs"></i> collection of rules](https://semgrep.dev/r) that you can download. Semgrep offers a 
+<i class="fas fa-external-link fa-xs"></i> Secrets-specific ruleset(https://semgrep.dev/p/secrets).
+
+### Sources of rules
+
+The Semgrep Registry contains rules imported from various repositories, including non-Semgrep individuals or groups, such as Trail of Bits and GitLab. You can view a rule's `license` key to ensure the license meets your needs.
+
+## Revocation
+
+Revoking a secret makes it inactive. This can be done when a secret isn't required anymore or if a secret becomes compromised.
+
+## Rotation
+
+Rotating secrets is the process of updating a secret regularly. If a secret is leaked, regular rotation can ensure that the credential is valid only for a limited time. Rotating secrets can also minimize risk due to the reuse of secrets.
+
+## Ruleset
+
+Rulesets are rules related through a programming language, OWASP category, or framework. Rulesets are curated by the team at Semgrep and updated as new rules are added to the Semgrep Registry.
+
+## Scan target
```

Comment:
I think this can be componentized (shared with Code IIRC?) but not a priority for this PR.